File: src/main/java/org/radarbase/management/service/dto/DataLogDTO.java
package org.radarbase.management.service.dto;

import org.radarbase.management.domain.enumeration.DataGroupingType;

import javax.persistence.Column;
import java.time.Instant;

public class DataLogDTO {
    private DataGroupingType groupingType;

    private Instant time;

    public DataGroupingType getGroupingType() {
        return groupingType;
    }

    public void setGroupingType(DataGroupingType groupingType) {
        this.groupingType = groupingType;
    }

    public Instant getTime() {
        return time;
    }

    public void setTime(Instant time) {
        this.time = time;
    }

}

File: src/main/java/org/radarbase/management/service/dto/DataLogDTO.kt
package org.radarbase.management.service.dto;

import org.radarbase.management.domain.enumeration.DataGroupingType;
import com.fasterxml.jackson.annotation.JsonInclude

import javax.persistence.Column;
import java.time.Instant;




@JsonInclude(JsonInclude.Include.NON_NULL)
class DataLogDTO {
    var groupingType: DataGroupingType? = null;
    var time: Instant? = null;

    override fun toString(): String {
        return ("DataLogDTO{" + "groupingType=" + groupingType
                + ", time=" + time + '\''
                + '}')
    }




}
